docs/reference/api/storage/LittleFileSystem.md

```diff
@@ -1,11 +1,60 @@
 ## LittleFileSystem
 
-[Add description here.]
+The little filesystem (LittleFS) is a fail-safe filesystem designed for embedded systems, specifically for microcontrollers that use flash storage.
```

Comment:
what kind of flash storage or does it matter?

Reply:
Hmm, are you thinking we should add "external" to be specific?

Reply:
Would be preferred as that's what we'll position this for